Makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: build-garage-release run-garage-release
+
+GARAGE_RELEASE="v2018.10.0"
+
+build-garage-release: docker/Dockerfile.release docker/mjkey.txt
```

Comment:
nah this makes no sense. there is only one set of canonical Dockerfiles with a single set of names.

the ones in the release branch are compliant with the release. the ones on the master branch are compliant with master.

Dockerfiles are code, just like any other Python file. we don't have -release and -local versions of logger.py, do we?

Reply:
Got it! Once this PR is approved, I will back port this commit to the 2018.10 release to build the corresponding version images.

Reply:
I honestly don't think that's necessary. 2018.10 release only guarantees bug fixes, not new features.